components/CarouselWithImages.vue

```diff
@@ -0,0 +1,58 @@
+<script setup>
+const props = defineProps({
+  images: {
+    type: Array,
+    required: true
+  }
+})
```

Comment:
Maybe props destructuring for simplicity? Example:

 const { images } = props;

and then in line 11 and 19 use only images from const. Will it work?

Reply:
@PiotrDrechsler Good point, probably it'll work, I'll check.